server/src/main/java/org/apache/cassandra/sidecar/db/ConfigAccessorFactory.java

```diff
@@ -25,29 +27,22 @@
  */
 public class ConfigAccessorFactory
 {
-    private final KafkaConfigAccessor kafkaConfigAccessor;
-    private final CdcConfigAccessor cdcConfigAccessor;
+    private final Map<Service, ConfigAccessor> configAccessors = new HashMap<>();
 
     @Inject
     public ConfigAccessorFactory(KafkaConfigAccessor kafkaConfigAccessor,
                                  CdcConfigAccessor cdcConfigAccessor)
     {
-        this.kafkaConfigAccessor = kafkaConfigAccessor;
-        this.cdcConfigAccessor = cdcConfigAccessor;
+        configAccessors.put(Service.KAFKA, kafkaConfigAccessor);
+        configAccessors.put(Service.CDC, cdcConfigAccessor);
```

Comment:
create immutable map. 

```suggestion
        configAccessors = Map.of(Service.KAFKA, kafkaConfigAccessor,
                                 Service.CDC, cdcConfigAccessor);
```